Description: Removes sphinx.ext.intersphinx fron upstream conf.py
 Avoid network when building sphinx doc.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-14

--- python-tuskarclient-0.1.1.orig/doc/source/conf.py
+++ python-tuskarclient-0.1.1/doc/source/conf.py
@@ -80,7 +80,7 @@ gen_ref("v1", "Version 1 API Reference",
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx']
+extensions = ['sphinx.ext.autodoc']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
